pkgs/by-name/fw/fw-fanctrl/package.nix

```
@@ -0,0 +1,61 @@
+{
+  lib,
+  python3Packages,
+  python3,
+  fw-ectool,
+  fetchFromGitHub,
+}:
+
+let
+  setuptools_75_8_0 = python3Packages.setuptools.overrideAttrs (old: rec {
+    version = "75.8.0";
+    src = fetchFromGitHub {
+      owner = "pypa";
+      repo = "setuptools";
+      rev = "v${version}";
+      hash = "sha256-dSzsj0lnsc1Y+D/N0cnAPbS/ZYb+qC41b/KfPmL1zI4=";
+    };
+    patches = [ ];
+  });
+in
+python3Packages.buildPythonPackage rec {
+  pname = "fw-fanctrl";
+  version = "0-unstable-2025-03-09";
+
+  src = fetchFromGitHub {
+    owner = "TamtamHero";
+    repo = "fw-fanctrl";
+    hash = "sha256-c+/wEMhWZknEad6Gae6ZYjz2DtE5kkp/rnIOXb8TYeA=";
+    rev = "main";
```

```suggestion
    rev = "main";
    hash = "sha256-c+/wEMhWZknEad6Gae6ZYjz2DtE5kkp/rnIOXb8TYeA=";
```
we must pin a digest